src/Storages/StorageReplicatedMergeTree.cpp

```diff
@@ -1778,21 +1849,20 @@ bool StorageReplicatedMergeTree::executeFetch(LogEntry & entry)
     const auto storage_settings_ptr = getSettings();
     auto metadata_snapshot = getInMemoryMetadataPtr();
 
-    if (storage_settings_ptr->replicated_max_parallel_fetches && total_fetches >= storage_settings_ptr->replicated_max_parallel_fetches)
-    {
-        throw Exception("Too many total fetches from replicas, maximum: " + storage_settings_ptr->replicated_max_parallel_fetches.toString(),
+    if (storage_settings_ptr->replicated_max_parallel_fetches &&
+        total_fetches >= storage_settings_ptr->replicated_max_parallel_fetches)
+        throw Exception("Too many total fetches from replicas, maximum: " +
```

Comment:
If you are here and changing code formatting, could you please also replace
`throw Exception("Some msg " + agr1 + " lorem ipsum " + arg2, ERROR_CODE)`
with
`throw Exception(ERROR_CODE, "Some msg {} lorem ipsum {}", arg1, arg2)`
?